tests/DCPS/Compiler/xcdr/xcdr.idl

```diff
@@ -72,6 +72,27 @@ struct MutableXcdr12Struct {
   COMMON_FIELDS
 };
 
+enum CommonFields {
```

Comment:
This name could be confusing given it has nothing to do with the `COMMON_FIELDS` macro. Maybe something like `UnionDisc`?